UefiPayloadPkg/Library/CbParseLib/CbParseLib.c

```diff
@@ -735,3 +737,219 @@ ParseTPMPPIInfo (
 
   return RETURN_SUCCESS;
 }
+
+STATIC
+CONST CHAR8 *
+GetRecoveryReasonString(
+  IN UINT8 code
+  )
+{
+  switch ((enum vb2_nv_recovery)code) {
+  /* 0x00 */ case VB2_RECOVERY_NOT_REQUESTED:

```

Comment:
Maybe the comment with hex codes should be at the end of line? Now it makes the switch hard to read